### DISCUSSION TITLE
Python API endpoint works in production but not on local with "vercel dev"

### DISCUSSION THREAD

**Discussion by Garross** [vercel/vercel#8140] [Help] [answered]
Hi all, 
We've gotten the python serverless api to work in production but it only works for one of us in local.

When I try to use the api locally I get: 

>  Building @vercel/python@latest:api/generate.py
Installing required dependencies...
> Built @vercel/python@latest:api/generate.py [11s]
using HTTP Handler
using HTTP Handler
<class 'OSError'>
Traceback (most recent call last):
  File "C:\Users\edti5\AppData\Local\co.zeit.fun\Cache\runtimes\python\bootstrap
.py", line 147, in <module>

The other developer is on Linux while I'm using Windows 10. 
We both have python installed but I'm worried that there may be a conflict with which version we're using? 
Or is it more to do with the 'OSerror'? 

**Extra Information**
`"vercel": "^27.0.1"
"next": "12.1.5"
`

we had to install the vercel CLI globally to login and try use this in the first place.

**Comment by Garross** [comment, marked as answer]
https://github.com/vercel/fun/issues/56

I have a feeling that this is related to my problem. And that installing WSL (https://docs.microsoft.com/en-us/windows/wsl/install) maybe the solution before vercel fixes this. Will write again if I can confirm.

**Reply by amyegan** [reply]
Hi @Garross. Thanks for sharing this workaround! The team is working on a solution, but WSL should get you unstuck for now.

**Reply by Garross** [reply]
Hey @amyegan thanks for the quick reply. Can confirm that WSL worked well for us. Also wanted to add that we're loving what the vercel team does!!

**Reply by dmarcucci** [reply]
Hi @amyegan,

I mentioned a native workaround I found [on this discussion](https://github.com/vercel/vercel/discussions/8266#discussioncomment-3541017), but also wanted to bring it up here for visibility.

Not sure if it would be a suitable permanent fix, but perhaps you can forward the info to the team so they can investigate.

Thanks.
